### ISSUE TITLE
new project diff

### ISSUE THREAD

**Issue by ericwinger** [dalehenrich/Rowan#85]
This no longer works without tode:
```
Rowan projectTools diff diffForProjectName: projectDef projectName.
```
What is the new smalltalk expression? 

https://github.com/dalehenrich/Rowan/blob/3b06f1ac13c02202dd8b04d0fb1d719e1b5e5bcd/src/tonel/Rowan-Tode/TDRowanProjectBrowser.class.st#L203

**Comment by dalehenrich** [comment]
@ericwinger what we talked about is (when we were getting the package diff enabled) that you'd collect all of the package diffs into a single collection ... or at least take a look at that possibility ...

this issue is also identical to issue #58 :) ... I'll close it since that issue predates our discussion about diffs ..

**Comment by dalehenrich** [comment]
I guess we should talk this over on Monday if you don't remember the conversation ...

**Comment by dalehenrich** [comment]
... or if it isn't feasible to list all package diffs for a project in one windows

**Comment by ericwinger** [comment]
Gotcha. I didn't remember, but I do now. I'll put that on my list.

**Comment by ericwinger** [comment]
Fixed in Jadeite 0.9.0
